test/e2e/storage/csi_objects.go

```diff
@@ -54,6 +54,7 @@ var (
 		"hostpathplugin":   "v0.4.0",
 		"csi-attacher":     "v0.4.0",
 		"csi-provisioner":  "v0.4.0",
+		"csi-snapshotter":  "v0.4.0",
 		"driver-registrar": "v0.4.0",
```

Comment:
Please update the test to use the latest container images "v0.4.1".

Reply:
@xing-yang Updated, PTAL